client/routes.js

```diff
@@ -22,6 +22,7 @@ class Routes extends Component {
         <Route path="/login" component={Login} />
         <Route path="/signup" component={Signup} />
         <Route exact path="/products" component={listProducts} />
+        <Route exact path="/products/add" component={addProduct} />
```

Comment:
@garrett-green just a note here, that the recommended restful route is "/products/new"
you can check an example here:https://gist.github.com/alexpchin/09939db6f81d654af06b